src/pages/home/home.scss

```diff
@@ -0,0 +1,77 @@
+@import "~theme/base/variables";
+
+.homeContainer {
+  height: 100%;
+}
+
+.navigationBar {
+  padding: 1.25em 0em;
+}
+
+.leftNavLinks {
+  margin-left: 4em;
+}
+
+.navLinks {
+  display: inline-block;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #179A6F;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 1rem;
+  margin-right: 2em;
+}
+
+.mainHeaderBackground {
+  background: url('assets/main-header-background.png') no-repeat;
```

Comment:
Using an image is ok, but what about doing it with css only!! One request less to the server!! I've created an example for you: https://codepen.io/crysfel/pen/gdVyjW

Support for linear gradient is great: https://caniuse.com/#search=linear-gradient

Reply:
Can we fix this later? Seems like a premature optimization no?

Reply:
We can stack backgrounds in CSS ;-)